core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -143,4 +144,23 @@ package object config {
       .internal()
       .stringConf
       .createWithDefaultString("AES/CTR/NoPadding")
+
+  private[spark] val DRIVER_BIND_ADDRESS = ConfigBuilder("spark.driver.bindAddress")
+    .doc("Address where to bind network listen sockets on the driver.")
+    .stringConf
+    .createWithDefault(Utils.localHostName())
```

Comment:
It should, as long as he doesn't set "spark.driver.bindAddress" - which is a new setting that is being added to override that behavior.